tidb-cloud/tidb-cloud-quickstart.md

```diff
@@ -76,15 +76,9 @@ In Chat2Query, you can either simply type `--` followed by your instructions to
 
 After running the queries, you can see the query logs and results immediately at the bottom of the page.
 
-## Step 3. Try Playground
+## Step 3. Load sample data
```

Comment:
I would suggest let new users try the interactive tutorials in Step3. 
## Step 3. Try Interactive Tutorials. 
Click the ? button on the right bottom of your screen. And click the "Interactive Tutorials". Select one of the pre-built sample datasets and explore TiDB Cloud's features and capabilities.